server/notification.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/thoas/go-funk"
+
+	"github.com/mattermost/mattermost-plugin-confluence/server/serializer"
+	"github.com/mattermost/mattermost-plugin-confluence/server/service"
+	"github.com/mattermost/mattermost-plugin-confluence/server/util"
+)
+
+type notification struct {
+	*Plugin
+}
+
+func (p *Plugin) getNotification() *notification {
+	return &notification{
+		p,
+	}
+}
+
+func (n *notification) SendConfluenceNotifications(event serializer.ConfluenceEventV2, eventType, botUserID, userID string) {
+	url := event.GetURL()
+	if url == "" {
+		return
+	}
+
+	spaceKey, pageID := n.extractSpaceKeyAndPageID(event, eventType)
+	if spaceKey == "" || pageID == "" {
+		return
+	}
+
+	post := event.GetNotificationPost(eventType, url, botUserID)
+	if post == nil {
+		return
+	}
+
+	subscriptionChannelIDs := n.getNotificationChannelIDs(url, spaceKey, pageID, eventType, userID)
+	for _, channelID := range subscriptionChannelIDs {
+		post.ChannelId = channelID
+		if _, err := n.API.CreatePost(post); err != nil {
+			n.API.LogError("Unable to create Post in Mattermost", "Error", err.Error())
+		}
+	}
+}
+
+func (n *notification) extractSpaceKeyAndPageID(event serializer.ConfluenceEventV2, eventType string) (string, string) {
+	var spaceKey, pageID string
+
+	switch {
+	case strings.Contains(eventType, Comment):
```

Comment:
Same as my question in antoher file. What's the content of `eventType` ? Can't we do a direct comparison?

Reply:
https://github.com/mattermost-community/mattermost-plugin-confluence/pull/127#discussion_r1987471640